src/main/java/com/firebolt/jdbc/connection/settings/FireboltProperties.java

```diff
@@ -1,27 +1,27 @@
 package com.firebolt.jdbc.connection.settings;
 
-import java.util.*;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-import java.util.stream.Collectors;
-
-import org.apache.commons.lang3.StringUtils;
-import org.apache.commons.lang3.tuple.Pair;
-
 import lombok.Builder;
 import lombok.CustomLog;
 import lombok.NonNull;
 import lombok.Value;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 @Value
 @Builder(toBuilder = true)
 @CustomLog
 public class FireboltProperties {
 
-	private static final Pattern DB_PATH_PATTERN = Pattern.compile("/([a-zA-Z0-9_*\\-]+)");
+	public static final String SYSTEM_ENGINE_NAME = "system";
```

Comment:
Unfortunately it is still in use in a couple of palces and I do not want to use null instead.

Reply:
We would need to remove any mention of the term `system` customer-exposed code.
Also, this constant is related to functionality that no longer works so we need to make sure we don't rely on it
Before new identity, we used to pass `system` as the engine_name in order to create a connection to a system engine.
For the new identity, this will simply not work. In case we still use this constant in other places, it can mean that we still rely on/test this functionality somewhere, which is invalid and should be fixed. I think we would need to remove it anyways